app/Http/Controllers/FileUploadController.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Storage;
+use App\Http\Requests;
+
+class FileUploadController extends Controller
+{
+    public function uploadFile(Request $request)
+    {
+        $file = $request->file('image');
+
+        $file_name = time() . "." . $file->getClientOriginalExtension();
+
+        $s3 = Storage::disk('s3');
+        $file_path = '/starapi/tests/' . $file_name;
+        $s3->put($file_path, file_get_contents($file), 'public');
```

Comment:
We need to store this in own collection `uploads` - how else would we be able to fetch the data from S3?

Each document should have `userId` and `fileUrl` attributes attached to it.